nnpdf_data/nnpdf_data/theory_cards/837.yaml

```diff
@@ -0,0 +1,48 @@
+ID: 837
+PTO: 2
+FNS: FONLL-B
```

Comment:
I believe that this theory card for NLO should change a but, for example it is missing the DAMPOWERb and DAMPOWERc entries

Reply:
PTO is wrong :sweat_smile: 

But DAMPPOWER are not present in the nnpdf theorycards. They will only be after #2028

Reply:
Yes indeed, I agree with that.

Sorry that I have not noticed the PTO 😂